packages/docs/src/styles/components/navigation.scss

```diff
@@ -79,7 +79,20 @@
 
         &:hover,
         &:visited:hover {
-          background-color: #{scale-color($vertical-nav-item__color--hover, $alpha: -95%)};
+          position: relative;
+          z-index: 1;
+
+          &:before {
+            z-index: -1;
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+            content: '';
+            background-color: #{$vertical-nav-item__color--hover};
+            filter: opacity(0.05);
+          }
         }
```

Comment:
@zarahzachz , I had to make an update here because I was getting compilation errors when trying to use the css vars with `scale-color`. Instead, I had to do a bit of a workaround.